third_party/amd/lib/TritonAMDGPUToLLVM/TargetInfo.cpp

```diff
@@ -245,4 +245,9 @@ void TargetInfo::assertFail(RewriterBase &rewriter, Location loc,
 
 int TargetInfo::getSharedAddressSpace() const { return 3; }
 
+bool TargetInfo::supportVectorizedAtomics() const {
+  // Not implemented - default to "false" for now.
+  return false;
```

Comment:
Perhaps return true here as vectorized atomics is supported on AMD, e.g, https://github.com/triton-lang/triton/pull/4925/ , but perhaps it doesn't matter as this predicate isn't used anywhere on the AMD path.

Reply:
Switched to `true`.